### PR TITLE
Provide copy button for code blocks

### LINKED ISSUE #23: provide a copy button for code examples in the readme/docs
### Description

It would be nice if there is a copy button in the docs and readme examples to make it easier for users to try the examples and play with the code. This should be easy to implement, maybe using a sphinx library. The docs are in the docs folder in the repo, I will let this issue open for new comers ;)

### PR DESCRIPTION
**Pull request by florianbussmann**
This closes #23

Example from the generated Sphinx HTML documentation after running `make docs`

image

### CODE REVIEW

**Comment by nidhaloff** [comment]
@florianbussmann Hi, thanks for joining. How did you test this or how did you make the screenshots? 

For some reason the docs won't build after merging the PR

**Comment by florianbussmann** [comment]
@nidhaloff I ran `make docs` and it opened the documentation it has bult in my browser where I then captured the attached screenshot above. Before pushing I also ran `python setup.py test` and got no errors.

From readthedocs.org I got the following [build log 13860780](https://readthedocs.org/projects/igel/builds/13860780/) and I guess my code formatter broke something here which I will revert now

```python
/home/docs/checkouts/readthedocs.org/user_builds/igel/envs/latest/bin/python -m sphinx -T -b html -d _build/doctrees -D language=en . _build/html
Running Sphinx v1.8.5

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/igel/envs/latest/lib/python3.7/site-packages/sphinx/config.py", line 368, in eval_config_file
    execfile_(filename, namespace)
  File "/home/docs/checkouts/readthedocs.org/user_builds/igel/envs/latest/lib/python3.7/site-packages/sphinx/util/pycompat.py", line 150, in execfile_
    exec_(code, _globals)
  File "/home/docs/checkouts/readthedocs.org/user_builds/igel/checkouts/latest/docs/conf.py", line 20, in <module>
    import igel
ModuleNotFoundError: No module named 'igel'

During handling of the above exception, another exception occurred
```

File: docs/conf.py [hunk 1]
```diff
@@ -17,11 +17,11 @@
 # relative to the documentation root, use os.path.abspath to make it
 # absolute, like shown here.
 #
+import igel
 import os
```
Comment (florianbussmann) [inline docs/conf.py:21]: This may broke it

**Comment by nidhaloff** [comment]
@florianbussmann alright, it would actually be better if you can open a new PR for this fix ;)

**Pull request merged**